z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -80,11 +80,10 @@ After you performed the previous steps successfully, you get one ready OpenShift
     - This server **must not** be any of the IBM Cloud Infrastructure Center nodes
     - You can use a single LPAR server or virtual machine
       - Disk with at least 20 GiB
-- **(Optional)** A Bastion server, a machine that is used to configure DNS and Load Balancer for the Red Hat OpenShift installation
-    - If you use your own existing DNS server and Load Balancer for the Red Hat OpenShift installation, the bastion server **is not** required.
-    - If you have external or existing DNS server, but no Load Balancer for the Red Hat Openshift installation, please set `os_dns_domain` property, and then use a separate YAML `configure-haproxy` to configure the HAProxy in bastion server.
-    - If you have existing Load Balancer, but no DNS server for the Red Hat Openshift installation, you can use a separate YAML `configure-dns` to configure the DNS server in bastion server.
-    - If you don't have any existing DNS server or Load Balancer for the Red Hat Openshift installation, you need to create one Linux server as the bastion server and run playbook to configure DNS server and Load Balancer. You can also use the same Linux server that runs Ansible.
+- **(Optional)** A Bastion server, a machine that is used for deploying and managing the OpenShift container platform cluster
```

Comment:
Why we change this part? The bastion server is still the machine used to configure DNS and Load Balancer for the Red Hat OpenShift installation. And use still has 3 choices to install their DNS and Load Balancer service. What the difference is user need to set the subnet DNS record as the DNS server. Along to the 3 choices, the DNS server can be bastion server or user's external DNS server.

Reply:
not remove it, just combine the first and second item. And leave two choices here in preparation, user can get details about DNS and LB in Step2.
```
- **(Optional)** A Bastion server, a machine that is used for deploying and managing the OpenShift container platform cluster
    - This server is required if you want to use your own bastion server to resolve your Nova servers.
      - You can use our playbook to automatic config the DNS and Load Balancer, check [Step2](#creation-of-the-cluster).
    - This server is optional if you could add DNS and Load Balancer into your external bastion server manually, you can refer [Add-DNS-HAProxy](docs/add-dns-haproxy.md) to update it.
    - The firewalld service should be enabled and running in bastion server.
```